pages/dkp/kommander/2.3/clusters/custom-domain-certificate/managed/index.md

```diff
@@ -0,0 +1,146 @@
+
+Use the API yaml to customize the domain (via the `hostname` field), the certificate (via the `issuerRef` or `certificateSecretRef` field), or both. For this, refer to the following examples:
+
+1.  Create or update and apply the `KommanderCluster` object with the wanted ingress. Remember to specify the cluster in the `kubeconfigRef` name field.
+
+    In this example, you can enter the custom domain in the `hostname` field, and an **issuer to be used by cert-manager** to issue a certificate for the domain in the `issuerRef` field.
+
+    ```yaml
+    cat <<EOF | kubectl apply -f -
+    apiVersion: kommander.mesosphere.io/v1beta1
+    kind: KommanderCluster
+    metadata:
+      name: <cluster_name>
+      namespace: <workspace_namespace>
+    spec:
+      kubeconfigRef:
+        name: <cluster_name>-kubeconfig
+      clusterRef:
+        capiCluster:
+          name: <cluster_name>
+    ingress:
+      hostname: <cluster_hostname>
+      issuerRef:
+        namespace: <issuer_namespace>
+        name: <issuer_name>
+    EOF
+    ```
+
+    In this example, you can enter the custom domain in the `hostname` field, and the secret in the `certificateSecretRef` field for **customized certificates created for your hostname**:
+
+    ```yaml
+    cat <<EOF | kubectl apply -f -
+    apiVersion: kommander.mesosphere.io/v1beta1
+    kind: KommanderCluster
+    metadata:
+      name: <cluster_name>
+      namespace: <workspace_namespace>
+    spec:
+      kubeconfigRef:
+        name: <cluster_name>-kubeconfig
+      clusterRef:
+        capiCluster:
+          name: <cluster_name>
+    ingress:
+      hostname: <cluster_hostname>
+      certificateSecretRef:
+        namespace: <secret_namespace>
+        name: <secret_name>
+    EOF
+    ```
+
+<!-- 1.  TODO: Dev - Are there other steps, is there some sort of confirmation message or output?  -->
+
+## Troubleshooting
+
+If you want to ensure the customization for a domain or a certificate is completed, or if you want to obtain more information in case the customization fails, call up a list of statuses for the `KommanderCluster`:
+
+<!-- 1.  TODO: DEV - provide command -->
+
+<!-- 1.  TODO: DEV - provide example output -->
+
+## Automatic certificate management (ACME)
+
+Your managed or attached cluster can be configured to automatically issue a trusted certificate for the configured custom domain and renew it before expiration.
+
+### Let's encrypt
+
+Set up a Let’s Encrypt certificate for the cluster ingress, to validate the certificate for the cluster. Because ACME does not set up a CA bundle, some of the platform applications must be customized to use the certificate created by the ACME issuer:
```

Comment:
I think here we should stress that Issuer has to be setup on the cluster that is being configured. As @floridoo suggests we can have a single page https://github.com/mesosphere/dcos-docs-site/pull/4553/files#r905076954 for management and attached clusters.

It is important to note that `KommanderCluster` configuration changes are applied on the management cluster, but the `Issuer` object must be created on the configured cluster. A special case is management cluster, when `KommanderCluster` and `Issuer` is on the same cluster. In case of attached clusters, the `KommanderCluster` object is stored on management cluster and `Issuer` on attached cluster.